bin/tbg

```diff
@@ -272,7 +272,15 @@ help()
 }
 
 #!/usr/bin/env bash
-initCall="$0 $*"
+#Check whether the provided path is relative or absolute and
+#convert it to an abolute path
+script=$0
+case $script in
+        /*) absScriptPath=$script;; #called using abs. path
+        ./*) absScriptPath=$PWD/${script#"./"};; #called using ./, remove it
```

Comment:
This could be simplified by using `realpath`. It is installed on most systems (we use).

Reply:
Admittedly, this is a much more elegant solution which should work assuming the system is running a linux distribution younger than ~2012!

I have incorporated it into the script.